examples/toolchain_v1alpha1_useraccount_cr.yaml

```diff
@@ -9,9 +9,9 @@ spec:
   nsTemplateSet:
     tierName: basic
     namespaces:
-      - type: ide
-        revision: ab12ef
-      - type: cicd
-        revision: 34efcd
+      - type: dev
+        revision: rev1
+      - type: code
+        revision: rev1
       - type: stage
-        revision: cdef56
+        revision: rev1
```

Comment:
what's wrong with hashes? isn't it what we'll use?

Reply:
With hashes we will not know which is first (old) and which is second (new).  with `rev1`, `rev2` it's more clearer. that's the idea.

Reply:
but this is not how it is supposed to work - we will use hashes, not some manually set revision strings. 
We had a lot of discussions about it several times and every time we agreed on using hashes taken from github commits.

Reply:
If idea of old v/s new revision not appealing to you folks then I will revert it.

Reply:
Yeah. The git commit hashes have some cons but they solve some important problems and this is why as Matous mentioned it was decided to use hashes.

Reply:
plus, we will always update to the latest revision of a template, so comparing the hashes works and is enough